File: C++/FuzzBuzz.cpp
#include <iostream>
using namespace std;
 int main(){
     for (int i = 1; i < 101;i++){
        if(i%3==0 && i%5 ==0) cout<<"FuzzBuzz"<<endl;
        else if(i%3==0 ) cout<<"Fuzz"<<endl;
        else if(i%5==0 ) cout<<"Buzz"<<endl;
        else  cout<<i<<endl;

     }
return 0;
 }

File: Java/FuzzBuzz.java
public class FuzzBuzz {
    public static void main(String[] args){
        for (int i = 1; i < 101;i++){
       if(i%3==0 && i%5==0){
        System.out.println("FuzzBuzz");
       }else if(i%3==0){
        System.out.println("Fuzz");
       }else if(i%5==0){
        System.out.println("Fuzz");
       }else{
        System.out.println(i);
       }
    }
    }
}
